upgrading/automated_upgrades.adoc

```diff
@@ -591,6 +591,25 @@ Ansible playbook. This is the playbook to use if you were deploying logging for
 the first time on an existing cluster, but is also used to upgrade existing
 logging deployments.
 
+. If you have any Elasticsearch SearchGuard indices in the following naming format, you need to delete and reseed the indices because the naming format has changed. 
+Elastic search might not work as expected unless you update the indices:
++
+----
+.searchguard.logging-es-*
+----
++
+.. Run the following command to delete the SearchGuard indices:
++
+----
+# oc exec -c elasticsearch <pod> -- es_util --query=.searchguard* -XDELETE
+----
++
+.. Run the following command to reseed the SearchGuard indices:
++
+----
+# for p in (oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); do oc exec -c elasticsearch <pod> -- es_seed_acl; done
```

Comment:
@mburke5678 This command is not executable. Could you please modify it to : 
 for p in $(oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); do oc exec -c elasticsearch $p -- es_seed_acl; done

Reply:
@jcantrill Please take a look at ^^  The `$p` is the pod name, so I can replace with `<pod>`?

`for p in $(oc get pods -l component=es -o jsonpath={.items[*].metadata.name}); do oc exec -c elasticsearch <pod> -- es_seed_acl; done`

Reply:
No.   because its a command so they can actually copy and run